app/app.py

```diff
@@ -0,0 +1,43 @@
+import os
+# Use the package we installed
+from slack_bolt import App
+import logging
+import os
+# Import WebClient from Python SDK (github.com/slackapi/python-slack-sdk)
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+
+# Initializes your app with your bot token and signing secret
+app = App(
+    token=os.environ.get("SLACK_BOT_TOKEN"),
+    signing_secret=os.environ.get("SLACK_SIGNING_SECRET")
+)
+
+
+# WebClient instantiates a client that can call API methods
+# When using Bolt, you can use either `app.client` or the `client` passed to listeners.
+client = WebClient(token=os.environ.get("SLACK_BOT_TOKEN"))
+logger = logging.getLogger(__name__)
+
+
+
+
```

Comment:
Not expecting any of this to be done in this file; This file should only contain the functions or perhaps the utility class to send, receive and perform actions on slack